dev-support/Jenkinsfile

```diff
@@ -47,7 +47,7 @@ pipeline {
 
     options {
         buildDiscarder(logRotator(numToKeepStr: '5'))
-        timeout (time: 24, unit: 'HOURS')
+        timeout (time: 30, unit: 'HOURS')
```

Comment:
Because 24 hr is no longer sufficient for full QA run. Initial 5 builds timed out after 24 hr, hence this change.

Reply:
Could you separate jira to extend the timeout to backport it into lower branches easily?

Reply:
Done. Reverted the build timeout change. Created HADOOP-18142 to track extending timeout. Now that this change is reverted, we might not be able to get full QA results.
Thanks

Reply:
I have again put this change back until we get reliable QA results again. I will revert this before merge and after getting +1 for the rest of the changes.
Thanks